migrate/20240529_fix_broken_index.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  no_transaction
+
+  up do
+    alter_table(:billing_record) do
+      drop_index :resource_id, concurrently: true
+      add_index :resource_id, concurrently: true
```

Comment:
Whoah, this looks very weird :D Can we at least add a comment here about the old index's format being different?

Reply:
migration files are very rarely read and I added a detailed explanation to commit message. Is it OK to keep this as is?

Reply:
sure